# Copyright 2017 ZT Prentner IT GmbH
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU Affero General Public License as published
# by the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU Affero General Public License for more details.
#
# You should have received a copy of the GNU Affero General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.
#
# German translations for RKTool package.
msgid ""
msgstr ""
"Project-Id-Version: 0.1\n"
"POT-Creation-Date: 2017-02-21 16:04+CET\n"
"PO-Revision-Date: 2016-05-30 23:05+0200\n"
"Last-Translator: Mino Sharkhawy\n"
"Language-Team: German\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: cashreg.py:99
msgid "Invalid turnover counter size."
msgstr "Umsatzzähler-Länge ungültig."

#: cashreg.py:138 main.py:365 receipt.py:87 rktool.py:365
msgid "Invalid key."
msgstr "Ungültiger Schlüssel."

#: demo.py:83
msgid "The number of receipts must be at least 1."
msgstr "Die Anzahl der Rechnungen muss mindestens 1 sein."

#: main.py:165 rktool.py:165
msgid "Error"
msgstr "Fehler"

#: main.py:232 main.py:307 rktool.py:232 rktool.py:307
msgid "Valid Signature"
msgstr "Gültige Signatur"

#: main.py:237 rktool.py:237
msgid "No Signature"
msgstr "Keine Signatur"

#: main.py:269 rktool.py:269
msgid "Signature system broken"
msgstr "Sicherheitseinrichtung ausgefallen"

#: main.py:271 rktool.py:271
msgid "ZDA ID"
msgstr "ZDA ID"

#: main.py:272 rktool.py:272
msgid "Cash Register ID"
msgstr "Kassen-ID"

#: main.py:273 rktool.py:273
msgid "Receipt ID"
msgstr "Belegnummer"

#: main.py:274 rktool.py:274
msgid "Timestamp"
msgstr "Datum & Uhrzeit"

#: main.py:275 rktool.py:275
msgid "Sum Tax Normal"
msgstr "Betrag Satz Normal"

#: main.py:276 rktool.py:276
msgid "Sum Tax Reduced 1"
msgstr "Betrag Satz Ermäßigt 1"

#: main.py:277 rktool.py:277
msgid "Sum Tax Reduced 2"
msgstr "Betrag Satz Ermäßigt 2"

#: main.py:278 rktool.py:278
msgid "Sum Tax Zero"
msgstr "Betrag Satz Null"

#: main.py:279 rktool.py:279
msgid "Sum Tax Special"
msgstr "Betrag Satz Besonders"

#: main.py:280 rktool.py:280
msgid "Turnover Counter"
msgstr "Umsatzzähler"

#: main.py:281 rktool.py:281
msgid "Certificate Serial/Key ID"
msgstr "Zertifikat-Seriennummer/Schlüssel ID"

#: main.py:282 rktool.py:282
msgid "Chaining Value"
msgstr "Verkettungswert"

#: main.py:283 rktool.py:283
msgid "Signature"
msgstr "Signatur"

#: main.py:288 main.py:630 rktool.py:288 rktool.py:630
msgid "Verifying..."
msgstr "Überprüfe..."

#: main.py:301 main.py:611 main.py:646 rktool.kv:145 rktool.kv:248
#: rktool.py:301 rktool.py:611 rktool.py:646
msgid "Verify"
msgstr "Überprüfen"

#: main.py:328 main.py:699 rktool.kv:233 rktool.py:328 rktool.py:699
msgid "Load AES Key"
msgstr "AES Schlüssel laden"

#: main.py:349 main.py:719 main.py:868 rktool.py:349 rktool.py:719
#: rktool.py:868
msgid "Malformed crypto container"
msgstr "Ungültiges Schlüsselmaterial"

#: main.py:410 rktool.py:410
msgid "No image taken."
msgstr "Kein Bild aufgenommen."

#: main.py:419 rktool.py:419
msgid "Failed to compress image."
msgstr "Fehler beim Komprimieren des Bildes."

#: main.py:438 main.py:459 rktool.py:438 rktool.py:459
msgid "No QR codes found."
msgstr "Keine QR-Codes gefunden."

#: main.py:446 rktool.py:446
msgid "Load Receipt"
msgstr "Rechnung laden"

#: main.py:559 rktool.py:559
msgid "Group %d"
msgstr "Gruppe %d"

#: main.py:571 main.py:578 main.py:762 rktool.py:571 rktool.py:578
#: rktool.py:762
msgid "Serial: "
msgstr "Seriennummer: "

#: main.py:653 rktool.py:653
msgid "Valid DEP"
msgstr "Gültiges DEP"

#: main.py:671 rktool.kv:230 rktool.py:671
msgid "Load DEP"
msgstr "DEP laden"

#: main.py:730 rktool.py:730
msgid "Public Keys"
msgstr "Öffentliche Schlüssel"

#: main.py:732 rktool.py:732
msgid "Certificates"
msgstr "Zertifikate"

#: main.py:767 rktool.py:767
msgid "Key ID: "
msgstr "Schlüssel ID: "

#: main.py:780 rktool.py:780
msgid "Load PEM Public Key"
msgstr "Öffentlichen Schlüssel (PEM) laden"

#: main.py:785 rktool.py:785
msgid "Load PEM Certificate"
msgstr "Zertifikat (PEM) laden "

#: main.py:806 rktool.py:806
msgid "Enter Public Key ID"
msgstr "ID des öffentlichen Schlüssels eingeben"

#: main.py:836 rktool.py:836
msgid "Load Key Store"
msgstr "Schlüsselspeicher laden"

#: main.py:842 rktool.py:842
msgid "Save Key Store"
msgstr "Schlüsselspeicher speichern"

#: receipt.py:43 verify.py:93
msgid "At receipt \"{0}\": {1}"
msgstr "Bei Rechnung \"{0}\": {1}"

#: receipt.py:62
msgid "Malformed receipt."
msgstr "Ungültige Rechnung."

#: receipt.py:71
msgid "Unknown algorithm."
msgstr "Unbekannter Algorithmus."

#: receipt.py:79
msgid "Algorithm mismatch."
msgstr "Inkompatible Algorithmen."

#: receipt.py:116
msgid "Unknown Receipt"
msgstr "Unbekannte Rechnung."

#: receipt.py:245 receipt.py:359 receipt.py:414 receipt.py:510
msgid "You need to sign the receipt first."
msgstr "Die Rechnung muss zuerst signiert werden."

#: receipt.py:550
msgid "Can't decrypt turnover counter, this is a dummy receipt."
msgstr "Kann Umsatzzähler nicht entschlüsseln. Dies ist ein Trainingsbeleg."

#: receipt.py:552
msgid "Can't decrypt turnover counter, this is a reversal receipt."
msgstr "Kann Umsatzzähler nicht entschlüsseln. Dies ist ein Stornobeleg."

#: receipt.py:588 verify_receipt.py:419
msgid "Input format must be one of %s."
msgstr "Eingabeformat muss eines aus %s sein."

#: receipt.py:592
msgid "Output format must be one of %s."
msgstr "Ausgabeformat muss eines aus %s sein."

#: rktool.kv:52 rktool.kv:75 rktool.kv:93
msgid "Cancel"
msgstr "Abbrechen"

#: rktool.kv:55
msgid "Load"
msgstr "Laden"

#: rktool.kv:78
msgid "Save"
msgstr "Speichern"

#: rktool.kv:96
msgid "Ok"
msgstr "Ok"

#: rktool.kv:128
msgid "Receipt"
msgstr "Rechnung"

#: rktool.kv:132 rktool.kv:286
msgid "Exit"
msgstr "Beenden"

#: rktool.kv:149
msgid "Decrypt Turnover Counter"
msgstr "Umsatzzähler entschlüsseln"

#: rktool.kv:197
msgid "JWS string"
msgstr "JWS-Zeichenkette"

#: rktool.kv:203
msgid "QR code"
msgstr "QR-Code"

#: rktool.kv:208
msgid "OCR code"
msgstr "OCR-Code"

#: rktool.kv:213
msgid "URL"
msgstr "URL"

#: rktool.kv:217
msgid "Read Receipt"
msgstr "Rechnung lesen"

#: rktool.kv:268
msgid "Import"
msgstr "Importieren"

#: rktool.kv:271
msgid "Export"
msgstr "Exportieren"

#: rktool.kv:279
msgid "Registrierkassen Tool"
msgstr "Registrierkassen Tool"

#: rktool.kv:297
msgid "Verify Receipt"
msgstr "Rechnung überprüfen"

#: rktool.kv:301
msgid "Verify DEP"
msgstr "DEP überprüfen"

#: rktool.kv:305
msgid "Key Store"
msgstr "Schlüsselspeicher"

#: run_test.py:64
msgid "Need key material for {} signature devices, got {} key pairs."
msgstr ""
"Brauche Schlüsselmaterial für {} Sicherheitseinrichtungen, habe {} "
"Schlüsselpaare bekommen."

#: run_test.py:270
msgid "I need keys and certificates for %d signature devices."
msgstr "Ich brauche Schlüssel und Zertifikate für %d Sicherheitseinrichtungen."

#: run_test.py:282 test_verify.py:256
msgid "Turnover counter size needs to be between 5 and 16."
msgstr "Umsatzzähler-Länge muss zwischen 5 und 16 sein."

#: sigsys.py:154
msgid "Selected algorithm not supported."
msgstr "Gewählter Algorithmus wird nicht unterstützt."

#: test_verify.py:71 test_verify.py:74 test_verify.py:79
msgid "no Exception"
msgstr "kein Fehler"

#: test_verify.py:117
msgid "State update without verification failed."
msgstr "Zustandsupdate ohne Verifikation fehlgeschlagen."

#: test_verify.py:126
msgid "Expected {} in turnover counter but got {}."
msgstr "Erwartete {} im Umsatzzähler, aber habe {} bekommen."

#: test_verify.py:139
msgid "Expected \"{}\" but got \"{}\", message: \"{}\""
msgstr "Erwartete \"{}\", aber habe \"{}\" bekommen, Nachricht: \"{}\""

#: test_verify.py:146
msgid "Expected \"{}\" at receipt \"{}\" but it occured at \"{}\" instead"
msgstr ""
"Erwartete \"{}\" bei Rechnung \"{}\", aber habe es stattdessen bei \"{}\" "
"bekommen"

#: test_verify.py:182
msgid "Result mismatch: without parsing {}:>{}<, with parsing {}:>{}<"
msgstr "Ergebnisse verschieden: ohne parsing {}:>{}<, mit parsing {}:>{}<"

#: test_verify.py:217
msgid "No run label"
msgstr "Kein Simulationsname"

#: test_verify.py:234
msgid "{} tests run, {} failed, {} errors"
msgstr "{} Tests ausgeführt, {} fehlgeschlagen, {} Fehler"

#: verification_state.py:40
msgid "No cash register with index {}."
msgstr "Keine Kassa mit Index {}."

#: verification_state.py:45
msgid "The last cash register has no registered start receipt."
msgstr "Die letzte Kassa hat keinen registrierten Startbeleg."

#: verification_state.py:123
msgid "Start Receipt"
msgstr "Startbeleg"

#: verification_state.py:124
msgid "Last Receipt"
msgstr "Letzter Beleg"

#: verification_state.py:125
msgid "Last Turnover Counter"
msgstr "Letzter Umsatzzähler"

#: verification_state.py:126
msgid "Need Restore Receipt"
msgstr "Benötigt Nullbeleg"

#: verification_state.py:231
msgid "Cash Register {}:"
msgstr "Kassa {}:"

#: verification_state.py:234
msgid "Used Receipt IDs"
msgstr "Benutzte Belegnummern"

#: verify.py:55
msgid "Malformed DEP"
msgstr "Ungültiges DEP"

#: verify.py:64
msgid "Malformed certificate: \"{}\""
msgstr "Ungültiges Zertifikat: \"{}\""

#: verify.py:73
msgid "Element \"{}\" missing from DEP"
msgstr "Element \"{}\" fehlt in DEP"

#: verify.py:83
msgid "GGS Cluster is not supported in an open system."
msgstr "GGS Verbund wird in offenen Systemen nicht unterstützt."

#: verify.py:104
msgid "Previous receipt is not \"{0}\"."
msgstr "Vorherige Rechnung ist nicht \"{0}\"."

#: verify.py:115
msgid "Receipt after restored signature system must not have any turnover."
msgstr ""
"Rechnung nach Wiederherstellung der Sicherheitseinrichtung darf keinen "
"Umsatz haben."

#: verify.py:125
msgid "Receipt ID already in use."
msgstr "Rechnungs-ID wird bereits verwendet."

#: verify.py:134
msgid "Turnover counter invalid."
msgstr "Umsatzzähler ungültig."

#: verify.py:143
msgid "Register ID changed."
msgstr "Kassen-ID geändert."

#: verify.py:153
msgid "Receipt was created before previous receipt."
msgstr "Rechnung wurde vor vorhergehender Rechnung erstellt."

#: verify.py:163
msgid "The system type changed."
msgstr "Typ des Systems wurde geändert."

#: verify.py:173
msgid "The size of the turnover counter changed."
msgstr "Die Länge des Umsatzzählers wurde geändert."

#: verify.py:182
msgid "No certificate specified in DEP and multiple groups used."
msgstr "Kein Zertifikat angegeben, aber DEP hat mehrere Gruppen."

#: verify.py:193
msgid "Certificate \"%s\" is not trusted."
msgstr "Zertifikat \"%s\" wird nicht vertraut."

#: verify.py:204
msgid "Certificate \"{}\" was not signed by \"{}\"."
msgstr "Zertifikat \"{}\" wurde nicht von \"{}\" signiert."

#: verify.py:216
msgid ""
"Two certificates with serial \"{0}\" detected (fingerprints \"{1}\" and "
"\"{2}\"). This may be an attempted attack."
msgstr ""
"Zwei Zertifikate mit Seriennummer \"{0}\" gefunden (Fingerabdrücke \"{1}\" "
"und \"{2}\"). Dies könnte ein versuchter Angriff sein."

#: verify.py:225
msgid "Initial receipt not signed."
msgstr "Startbeleg nicht signiert."

#: verify.py:233
msgid "Initial receipt has nonzero turnover."
msgstr "Startbeleg hat Umsatz ungleich Null."

#: verify.py:243
msgid "Initial receipt has not been chained to the cash register ID."
msgstr "Startbeleg ist nicht mit der Kassen-ID verkettet."

#: verify.py:253
msgid ""
"Initial receipt in cluster has not been chained to the previous cash "
"register's initial receipt."
msgstr ""
"Startbeleg in Verbund ist nicht mit dem Startbeleg der vorherigen Kassa "
"verkettet."

#: verify.py:262
msgid "Initial receipt is a dummy or reversal receipt."
msgstr "Startbeleg ist ein Trainings- oder Storno-Beleg."

#: verify.py:976
msgid "Verification successful."
msgstr "Überprüfung erfolgreich."

#: verify_receipt.py:42
msgid "Certificate serial mismatch."
msgstr "Zertifikat-Seriennummern stimmen nicht überein."

#: verify_receipt.py:49
msgid "Certificate serial invalid."
msgstr "Zertifikat-Seriennummer ungültig."

#: verify_receipt.py:57
msgid "No public key found."
msgstr "Kein öffentlicher Schlüssel gefunden."

#: verify_receipt.py:64
msgid "Invalid Signature."
msgstr "Ungültige Signatur."

#: verify_receipt.py:73
msgid "{} This is probably fine."
msgstr "{} Das ist wahrscheinlich in Ordnung."

#: verify_receipt.py:81
msgid "Signature System failed."
msgstr "Sicherheitseinrichtung ausgefallen."

#: verify_receipt.py:89
msgid "Invalid URL hash."
msgstr "Ungültige URL Prüfsumme."

#: verify_receipt.py:98
msgid "Invalid certificate provider."
msgstr "Ungültiger Zertifizierungsdiensteanbieter."

#: verify_receipt.py:106
msgid "Null receipt not signed."
msgstr "Nullbeleg nicht signiert."

#: verify_receipt.py:441
msgid "Line {: >3}: {}"
msgstr "Zeile {: >3}: {}"

#: verify_receipt.py:444
msgid "{} of {} receipts verified successfully."
msgstr "{} von {} Rechnungen wurden erfolgreich überprüft."
